addons/website/static/src/scss/website.scss

```diff
@@ -190,6 +190,16 @@ $-seen-urls: ();
     }
 };
 
+html {
+    // In the line below, a white background is set on HTML due to the fallback
+    // iframe used in edit mode. This second iframe is placed behind the website
+    // iframe. And so, if the website BODY uses a transparent background color,
+    // the fallback iframe becomes visible. A white background on the html
+    // element prevents this issue.
+    // TODO: Remove if a better solution is found for the edit mode fallback.
```

Comment:
No need. "TODO this code should be changed if we change it" :wink: 

Maybe you can add "See `/website/iframefallback`" in your explication above, that way if we ever remove that double iframe, we'll remove the controller, we'll grep the route and we'll find this CSS.